src/firefox/ffBrowser.ts

```diff
@@ -46,6 +46,24 @@ export class FFBrowser extends BrowserBase {
       browser._defaultContext = new FFBrowserContext(browser, null, options.persistent);
       promises.push((browser._defaultContext as FFBrowserContext)._initialize());
     }
+    if (options.proxy) {
+      const proxyServer = new URL(options.proxy.server);
+      let aType: 'http'|'https'|'socks'|'socks4' = 'http';
+      if (proxyServer.protocol === 'socks5:')
+        aType = 'socks';
+      else if (proxyServer.protocol === 'socks4:')
+        aType = 'socks4';
+      else if (proxyServer.protocol === 'https:')
```

Comment:
Is it supported in other browsers? If no, let's update the docs.

Reply:
It does work in Chromium! WebKit is all different across platforms - I haven't checked yet.